recipes.js

```diff
@@ -8,3 +8,69 @@ mongoose.connect('mongodb://localhost/recipeApp')
   }).catch(err => {
     console.error('Error connecting to mongo', err);
   });
+
+
+const Recipe = mongoose.model('Recipe', recipeSchema);
```

Comment:
This line should come after your declare your recipeSchema, otherwise it will say that recipeSchema is not defined